backend/Application.UnitTests/Common/Constants.cs

```diff
@@ -10,8 +10,10 @@ public static class Constants
     public const string NewPassword = "Password";
     public const UserRole Role = UserRole.Admin;
     public const string StaffCode = "SD0001";
-    public const string FirstName = "Le";
-    public const string LastName = "Thanh Dat";
+    public const string FirstName = "Do";
+    public const string LastName = "Trung Dung";
+    public const string NewUserName = "dungdt";
+    public const string NewCreatePassword = "dungdt@05102002";
```

Comment:
You need to discuss with team to have the constant string like this instead set default as your name. GDPR policy